crates/daemon/src/wallet.rs

```diff
@@ -398,7 +451,7 @@ trait BuildLockTx {
     ) -> Result<PartiallySignedTransaction>;
 }
 
-impl<D> BuildLockTx for bdk::Wallet<D>
+impl<D> BuildLockTx for Wallet<D>
```

Comment:
🙃 Not your change, but to be consistent with other type parameters.

```suggestion
impl<DB> BuildLockTx for Wallet<DB>
```